packages/vc-verification/src/revocation-verification.ts

```diff
@@ -122,4 +146,61 @@ export class RevocationVerification {
       throw new InvalidRevokerType(role, revokers?.revokerType);
     }
   }
+
+  /**
+   * Checks the revocation status for the given issuer and role
+   * @param issuer issuer DID
+   * @param role namespace
+   * @returns
+   */
+  async checkRevocationStatus(issuer: string, role: string) {
```

Comment:
In my opinion, it is really nice to have the return types to make it easier to read the code
```suggestion
  async checkRevocationStatus(issuer: string, role: string): Promise<Boolean> {
```